fastlane/Gemfile.lock

```diff
@@ -217,6 +217,7 @@ GEM
 
 PLATFORMS
   arm64-darwin-22
+  x86_64-darwin-22
```

Comment:
Is this change needed? This seems to be the only thing different from the previous PR, thoughts?

Reply:
Thanks for explaining, @seanstrom i believe you're on an arm machine? If so could you pls run `make run-ios` and see if this part remains unchanged. If so then this change is good to keep. Else we'd need to dig into it further...

Reply:
I don't see that part change, but I have a `podfile.lock` change for some reason: https://github.com/status-im/status-mobile/commit/7bef8b0bade9fec768c1bec6281eae22d5773dbf

Reply:
Just checked and that `podfile.lock` change seems to be present on develop, so perhaps unrelated to these changes.

Reply:
Okay I triple checked by running `make clean` for develop and this branch, and then I ran `make run-ios` for each branch, and I see no diffs 🙌

Reply:
Thanks for verifying @seanstrom !!